base_external_api_ftp/models/apicli_connection_ftp.py

```diff
@@ -112,7 +112,7 @@ def _ftp_upload_directory(self, ftp_session, from_local_dir, to_server_dir):
                 ftp_session, os.path.join(from_local_dir, subdir), subdir
             )
 
-    def _send_ftp_upload(self, from_local_dir, to_server_dir="."):
+    def _send_ftp_upload(self, from_local_dir, to_server_dir="Inbox"):
```

Comment:
The "Inbox" is really a configuration, and should be passed to the cron method, rather than being set as a default in code.